### PR TITLE
Beautified `README.md` & Themed docs like `open-event-api-server`

### LINKED ISSUE #954: Beautify `README.md` & Theme the docs like `open-event-api-server`
- Imply the style which is currently used in `open-event-api-server` **repo**. :wink: :wine_glass: 
- Add `Project Title` :heart:
- Fix improper rendering of `github-badges`. :+1: 
- Link the `LICENSE.md` to `README.md` :heart:
- Fix typo(s) and prettify the `README.md`

### PR DESCRIPTION
**Pull request by 0x48piraj**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- Implied the style which is currently used in `open-event-api-server` **repo**.
- Added `Project Title` :heart:
- Fixed improper `github-badges`.
- Removed strings and Linked the `LICENSE.md` to `README.md` :heart:
- Bullet pointed `Communication` & Highlighted, Quoted URL(s)
- Fixed many typo(s) and used *italics*, **bold** to prettify the `README.md`

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-955-open-event-frontend-fossasia.surge.sh

**Comment by 0x48piraj** [comment]
Sorry @niranjan94 

_Can you please explain what you actually meant ?_ :confounded:

**Comment by sumedh123** [comment]
@0x48piraj Once your tests pass, the deployment link is generated automatically. That's why the link is given there.

**Comment by 0x48piraj** [comment]
Oh! It meant, all tests passed successfully :blush:
Thanks for telling me. :smile:

File: README.md [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
 [![Gitter](https://img.shields.io/badge/chat-on%20gitter-ff006f.svg?style=flat-square)](https://gitter.im/fossasia/open-event-frontend)
 [![Codacy grade](https://img.shields.io/codacy/grade/0d51cf60fc734d3699fd6eff6054e483.svg?style=flat-square)](https://www.codacy.com/app/niranjan94/open-event-frontend)
 [![codecov](https://img.shields.io/codecov/c/github/fossasia/open-event-frontend/development.svg?style=flat-square)](https://codecov.io/gh/fossasia/open-event-frontend)
-
 [![Greenkeeper badge](https://badges.greenkeeper.io/fossasia/open-event-frontend.svg)](https://greenkeeper.io/)
 [![Known Vulnerabilities](https://snyk.io/test/github/fossasia/open-event-frontend/badge.svg)](https://snyk.io/test/github/fossasia/open-event-frontend)
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/open-event-frontend/localized.svg)](http://translate.eventyay.com/project/open-event-frontend)
 
-> The front end for the Open Event API Server
+> The front end for the **Open Event API Server**
```
Comment (geekyd) [inline README.md]: Make this `Open Event Server`

File: README.md [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
 **API Documentation:**
-- Every installation of the Open Event API Server project includes API docs, (e.g. here on the test install [https://open-event-api.herokuapp.com](https://open-event-api.herokuapp.com)).
-- A hosted version of the API docs is available in the `gh-pages` branch of the Open Event API Server repository at [http://dev.eventyay.com/api/v1](http://dev.eventyay.com/api/v1)
+- Every installation of the **Open Event API Server** project includes API docs, (e.g. here on the test install [https://open-event-api.herokuapp.com](https://open-event-api.herokuapp.com)).
```
Comment (geekyd) [inline README.md]: Make this `Open Event Server`

File: README.md [hunk 3]
```diff
@@ -1,40 +1,44 @@
...
+- A hosted version of the API docs is available in the `gh-pages` branch of the **Open Event API Server** repository at [http://dev.eventyay.com/api/v1](http://dev.eventyay.com/api/v1)
```
Comment (geekyd) [inline README.md]: Make this `Open Event Server`

File: README.md (hunk 1 above)
Comment (0x48piraj) [reply README.md]: All changes made! Thanks @geekyd  😊

File: README.md (hunk 2 above)
Comment (0x48piraj) [reply README.md]: All changes made! Thanks @geekyd  😊

File: README.md (hunk 3 above)
Comment (0x48piraj) [reply README.md]: All changes made! Thanks @geekyd  😊

**Comment by niranjan94** [comment]
Deployment link: https://pr-955-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-955-open-event-frontend-fossasia.surge.sh

File: README.md [hunk 4]
```diff
@@ -103,4 +107,6 @@ This project has acceptance, integration and unit tests located inside the `test
 
 ## License
 
-This project is currently licensed under the Apache License version 2.0. A copy of `LICENSE` should be present along with the source code. To obtain the software under a different license, please contact [FOSSASIA](http://blog.fossasia.org/contact/).
+**This project is currently licensed under the [Apache License version 2.0](LICENSE).**
```
Comment (sumedh123) [inline README.md]: Isn't this the incorrect syntax?

File: README.md [hunk 5]
```diff
@@ -1,40 +1,44 @@
...
 
 You will need the following things properly installed on your computer.
 
-* [Git](https://git-scm.com/)
-* [Node.js](https://nodejs.org/) (with NPM v5.2x)
-* [Bower](https://bower.io/) - `npm install -g bower`
-* [Ember CLI](https://ember-cli.com/) - `npm install -g ember-cli`
-* [PhantomJS](http://phantomjs.org/) - `npm install -g phantomjs-prebuilt`
-* [check-node-version](https://www.npmjs.com/package/check-node-version) - `npm install -g check-node-version`
+* **[Git](https://git-scm.com/)**
```
Comment (CosmicCoder96) [inline README.md]: Please don't make these bold it starts to look cumbersome  :) 
image

File: README.md (hunk 4 above)
Comment (0x48piraj) [reply README.md]: I don't think so @sudheesh001 !
All the other `open-event-*` repo's are following the later one :wink:

File: README.md (hunk 5 above)
Comment (0x48piraj) [reply README.md]: But, these changes were approved by other two repo's :open_mouth:

File: README.md (hunk 4 above)
Comment (sudheesh001) [reply README.md]: Looks like you tagged the wrong person. @0x48piraj you probably meant to tag @sumedh123

File: README.md (hunk 5 above)
Comment (0x48piraj) [reply README.md]: @CosmicCoder96 You checked those repos ?

**Pull request merged**